content/docs/concepts/architecture.md

```diff
@@ -0,0 +1,328 @@
+
+![Network Service Venn Diagram](/img/concepts/architecture/ns_venn.svg)
+
+A Network Service is a collection of Connectivity, Security, and Observability features applied to traffic.
+
+Examples of Network Services would include:
+- A simple distributed vL3 that allows the workloads to communicate via IP, optionally with DNS service for that vL3
+- A Traditional Network Service like Istio, Linkerd, Consul, or Kuma running over a vL3.  This allows specific workloads to be admitted to that Service Mesh, independent of where they run.
+It also allows a single workload to connect to multiple Traditional Service Meshes.  This can allow a workload to connect both to a companies Service Mesh, and also to the Service Meshes of its partners simultaneously.
+
+More sophisticated features (IPS, etc) can be composed into Network Services to add additional Security and Observability Features.
+
+### Clients
+
+A {{<color "#0050EF" >}}Client{{< /color >}} in Network Service Mesh, sometimes also called a {{<color "#0050EF" >}}Network Service Client (NSC){{</color>}} is a workload that asks to be connected to a Network Service
+by name.  A {{<color "#0050EF" >}}Client{{< /color >}} is independently authenticated (currently by Spiffe ID), and must be authorized to be attached to attach to a Network Service.
+
+For each Network Service to which a {{<color "#0050EF" >}}Client{{< /color >}} wishes to be connected, in addition to the name of that Network Service and the identity of the Client,
+an optional set of 'labels' (key value pairs) may be provided.  These 'labels' may be used by Network Service for {{<color "#008A00" >}}Endpoint{{< /color >}} selection, or by {{<color "#008A00" >}}Endpoints{{< /color >}} themselves
+to influence how the {{<color "#008A00" >}}Endpoint{{< /color >}} provides service to the {{<color "#0050EF" >}}Client{{< /color >}}.
+
+A {{<color "#0050EF" >}}Client{{< /color >}} may be a:
+- Pod
+- VM
+- Physical Server
+
+### vWires
+
+That which connects a {{<color "#0050EF" >}}Client{{< /color >}} to an {{<color "#008A00" >}}Endpoint{{< /color >}} is a {{<color "#6A00FF" >}}vWire{{< /color >}} or {{<color "#6A00FF" >}}Virtual Wire{{< /color >}}.
+
+![vWire](/img/concepts/architecture/vWire.svg)
+
+The contract of a {{<color "#6A00FF" >}}vWire{{< /color >}} is:
+- A packet ingressing the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}} will egress at the {{<color "#008A00" >}}Endpoint{{< /color >}}
+- A packet ingressing the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#008A00" >}}Endpoint{{< /color >}} will egress the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}}
+- Only packets that ingressed the {{<color "#6A00FF" >}}vWire{{< /color >}} at the {{<color "#0050EF" >}}Client{{< /color >}} will egress at the {{<color "#008A00" >}}Endpoint{{< /color >}}
```

Comment:
This list does not look like a list in the web view.

Reply:
Fixed.